webdriver-spec.html

```diff
@@ -7190,7 +7190,8 @@
         </table>
 
       <p>When required to <dfn>dispatch a keyDown action</dfn> with arguments
-        <var>action object</var> and <var>input state</var> a <a>remote end</a>
+        <var>source id</var>, <var>action object</var>, <var>input state</var>
+        and <var>tick duration</var> a <a>remote end</a>
```

Comment:
Sorry for the delay, this slipped through the cracks.

Yes, @AutomatedTester the patch's intention is to make uniform the argument of the various dispatch functions. And for most of the other dispatch functions there are 4 arguments, `tick duration` included.